includes/payment-methods/class-wc-stripe-express-checkout-helper.php

```diff
@@ -314,6 +314,22 @@ public function get_allowed_shipping_countries() {
 		return array_keys( $allowed_shipping_countries );
 	}
 
+	/**
+	 * Get the number of decimals supported by Stripe for the currency.
+	 *
+	 * @return int
+	 */
+	public static function get_stripe_currency_decimals() {
+		$currency = get_woocommerce_currency();
+		if ( in_array( $currency, WC_Stripe_Helper::no_decimal_currencies(), true ) ) {
```

Comment:
Should we convert `$currency` to lowercase to be safe?

In my test store `get_woocommerce_currency()` returns the currency in uppercase, but currencies in our helper functions are in lowercase.

Reply:
Done in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4480/commits/ec499636306b9b8c4bc7807a2e1042386e858576